src/renderer/components/cluster-settings/cluster-settings.tsx

```diff
@@ -0,0 +1,130 @@
+import React from "react";
+import { clusterStore } from "../../../common/cluster-store";
+import { ClusterProxySetting } from "./components/cluster-proxy-setting";
+import { ClusterNameSetting } from "./components/cluster-name-setting";
+import { ClusterHomeDirSetting } from "./components/cluster-home-dir-setting";
+import { ClusterAccessibleNamespaces } from "./components/cluster-accessible-namespaces";
+import { ClusterMetricsSetting } from "./components/cluster-metrics-setting";
+import { ShowMetricsSetting } from "./components/show-metrics";
+import { ClusterPrometheusSetting } from "./components/cluster-prometheus-setting";
+import { ClusterKubeconfig } from "./components/cluster-kubeconfig";
+import { entitySettingRegistry } from "../../../extensions/registries";
+import { CatalogEntity } from "../../api/catalog-entity";
+
+entitySettingRegistry.add([
+  {
+    apiVersions: ["entity.k8slens.dev/v1alpha1"],
+    kind: "KubernetesCluster",
+    source: "local",
+    title: "General",
+    components: {
+      View: (props: { entity: CatalogEntity }) => {
+        const cluster = clusterStore.enabledClustersList.find((cluster) => cluster.id === props.entity.metadata.uid);
+
+        if (!cluster) {
+          return null;
+        }
```

Comment:
I think it would be better to do the filter on enabled after finding by ID. Easier to refactor later (when we remove enabled) and more performant in the meantime.
```suggestion
        const cluster = clusterStore.getById(props.entity.metadata.uid);

        if (!cluster?.isEnabled) {
          return null;
        }
```

Reply:
Refactored.